can/interfaces/socketcan/socketcan_native.py

```diff
@@ -308,7 +338,7 @@ def captureMessage(sock):
     """
     # Fetching the Arb ID, DLC and Data
     try:
-        cf, addr = sock.recvfrom(can_frame_size)
+        cf, addr = sock.recvfrom(CAN_FRAME_HEADER_STRUCT.size + 64)
```

Comment:
Maybe pull this magic number out?

`CANFD_MTU`?

Reply:
Good idea.